frontend/src/pages/projects/screens/detail/storage/StorageTableRow.tsx

```diff
@@ -73,16 +65,8 @@ const StorageTableRow: React.FC<StorageTableRowProps> = ({
   }
 
   return (
-    <Tbody isExpanded={isExpanded}>
+    <Tbody>
```

Comment:
Since you removed the expandable section, you don't need the `Tbody` anymore. And don't forget to remove `disableRowRenderSupport` here https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/pages/projects/screens/detail/storage/StorageTable.tsx#L79